tutorials/napps/switch_l3_part2.rst

```diff
@@ -655,3 +670,9 @@ Good job!
 
 .. |dev_env| replace:: *Development Environment*
 .. _dev_env: http://tutorials.kytos.io/napps/development_environment_setup/
+
+.. |napps_server| replace:: *NApps Server*
+.. _napps_server: http://napps.kytos.io
+
+.. |napps_server_sign_up| replace:: **sign_up**
+.. _napps_server_sing_up: https://napps.kytos.io/signup/
```

Comment:
s/_napps_server_sing_up/_napps_server_sign_up/ - here and in the other footnotes.